### DISCUSSION TITLE
RF-Gateway (TEQ1870610) not connected

### DISCUSSION THREAD

**Discussion by Rantanplans** [OpenCCU/OpenCCU#2865] [🇩🇪 Fragen&Antworten]
Hallo,
ein Lan Gateway auf RaspberryMatic 3.77.7.20240826 angelernt und eingerichtet. Alles läuft soweit.
Leider bricht die Verbindung zum Gateway nach ein paar Stunden mit der Fehlermeldung (siehe Titel) ab.
Das Gatway ist weiterhin im Netzwerk sichtbar.
Ein Reboot der RaspberryMatic SW hilft. Danach ist das Gateway wieder für ein paar Stunden sichtbar. Danach wieder das gleiche Problem.
Die RaspberryMatic läuft auf einem Rasberry. Meine Router ist eine Fritz Box und das Homematic Lan Gateway hängt auf einem Fritz Repeater. Das Fritz Mesh Netz ist aktiv. Der Raspberry und das Lan Gateway sind über Lan an FritzBox und Repeater angebunden.

Hat jemand eine Idee. wie ich das Problem weiter eingrenzen kann?
Gruß und Dank voraus.

**Comment by Baxxy13** [comment]
> Hat jemand eine Idee. wie ich das Problem weiter eingrenzen kann?

Das LGW auch mal per Kabel an die Fritzbox hängen an der auch die RM hängt.
Verschwindet das Problem dann weißt du das es durch dein "Fritz Mesh Netz" hervorgerufen wird.

**Comment by Rantanplans** [comment]
Sorry, da hätte ich auch selbst drauf kommen können. Direkt auf der Fritzbox habe ich zwar auch Verbindungsabbrüche, aber das Gateway ist immer noch in der Homatic sichtbar und funktioniert noch
